fedimint-core/src/bitcoin_migration.rs

```diff
@@ -112,6 +112,74 @@ pub fn bitcoin30_to_bitcoin29_outpoint(outpoint: bitcoin30::OutPoint) -> bitcoin
     }
 }
 
+pub fn bitcoin29_to_bitcoin30_witness(witness: &bitcoin::Witness) -> bitcoin30::Witness {
+    bincode::deserialize(
+        &bincode::serialize(witness).expect("Failed to serialize bitcoin v29 witness"),
+    )
+    .expect("Failed to deserialize bitcoin v29 witness to bitcoin v30 witness")
+}
+
+pub fn bitcoin30_to_bitcoin29_witness(witness: &bitcoin30::Witness) -> bitcoin::Witness {
+    bincode::deserialize(
+        &bincode::serialize(witness).expect("Failed to serialize bitcoin v30 witness"),
+    )
+    .expect("Failed to deserialize bitcoin v30 witness to bitcoin v29 witness")
+}
```

Comment:
![Screenshot 2024-04-04 at 23-20-54 Witness in bitcoin blockdata witness - Rust](https://github.com/fedimint/fedimint/assets/84478054/90243fd0-80b4-4cd2-a7b6-263668d69836)
![Screenshot 2024-04-04 at 23-20-06 Witness in bitcoin blockdata witness - Rust](https://github.com/fedimint/fedimint/assets/84478054/a086a9e2-d13b-4109-bc8d-a8eeb5ad3d11)

Reply:
Done.